packages/eas-build-job/src/android.ts

```diff
@@ -107,6 +107,10 @@ export interface Job {
     prebuildCommand?: string;
   };
   expoBuildUrl?: string;
+  gitHubTriggerOptions?: {
```

Comment:
```suggestion
  githubTriggerOptions?: {
```

And this is why I love Zod.